common/issues_utils.py

```diff
@@ -31,8 +31,8 @@ def add_issue(issue):
     Fetch the current plugin issues.yaml if it exists and add new issue with
     description of the issue.
     """
-    if not os.path.isdir(PLUGIN_TMP_DIR):
-        raise Exception("plugin tmp dir  '{}' not found".
+    if PLUGIN_TMP_DIR is None or not os.path.isdir(PLUGIN_TMP_DIR):
```

Comment:
This should never be None. If it is failing because of a unit test you need to make sure it gets set in the unit test. See https://github.com/dosaboy/hotsos/blob/master/tests/unit/test_storage.py#L21 for an example of how to do this.

Reply:
Fixed